Content.Server/Chemistry/ReagentEffectConditions/ReagentThreshold.cs

```diff
@@ -18,7 +18,7 @@ public class ReagentThreshold : ReagentEffectCondition
         [DataField("max")]
         public FixedPoint2 Max = FixedPoint2.MaxValue;
 
-        public override bool Condition(IEntity solutionEntity, Solution.ReagentQuantity reagent)
+        public override bool Condition(IEntity solutionEntity, IEntity organEntity, Solution.ReagentQuantity reagent)
```

Comment:
TBD using EntityUid's here I don't envy you

Reply:
if i do this should i just. pass in entitymanager too or something

Reply:
im gonna do that